docs/api/rest_api_usage.md

```diff
@@ -0,0 +1,697 @@
+
+| Code  | Message                | Description                                                                                                                                                                                                                                                  |
+|-------|------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `200` | OK                     | The resource has been found.                                                                                                                                                                                                                                 |
+| `201` | Created                | The request to create a new item has succeeded; the response `Location` header indicates when the created item could be consulted.                                                                                                                           |
+| `204` | No Content             | The request has succeeded and there is nothing to say about it in the response header nor body (for example when publishing or deleting).                                                                                                                    |
+| `301` | Moved Permanently      | The resource should not be accessed this way; the response `Location` header indicates the proper way.                                                                                                                                                       |
+| `307` | Temporary Redirect     | The resource is available at another URL considered as its main; the response `Location` header indicates this main URL.                                                                                                                                     |
+| `400` | Bad Request            | The input (payload) doesn't have the proper schema for the resource.                                                                                                                                                                                         |
+| `401` | Unauthorized           | The user does not have the permission to make this request.                                                                                                                                                                                                  |
+| `403` | Forbidden              | The user has the permission but action can't be done because of repository logic (for example, when trying to create an item with an already existing ID or identifier, when attempting to update a version in another state than draft).                    |
```

Comment:
```suggestion
| `403` | Forbidden              | The user has the permission but action can't be performed because of Repository logic (for example, when trying to create an item with an already existing ID or identifier, when attempting to update a version in another state than draft).                    |
```